firmware/src/hal/src/x86/communication.c

```diff
@@ -22,404 +22,23 @@
  * IN THE SOFTWARE.
  */
 
-#include <stdio.h>
```

Comment:
How about moving this to `firmware/src/hal/src/common/communication.c` since it is now completely platform-independent?

Reply:
We are actually currently just using symlinks to the `firmware/src/hal/src/common` directory, so will do that. I won't be removing symlinks altogether since what is going to happen for SGX is that we will share some libraries between platforms, but not between all platforms (e.g., we might share `keccak256` between SGX and x86, but Ledger will have its own). So most likely the `common` directory will just have to go.